config/esp32/components/chip/idf_component.yml

```diff
@@ -25,3 +25,9 @@ dependencies:
         rules:
             - if: "idf_version >=5.0"
             - if: "target != esp32h2"
+
+    espressif/esp_delta_ota:
+        version: "1.1.0"
```

Comment:
Why a specific version and not the latest after the lowest supported?

Reply:
Updated